### DISCUSSION TITLE
Best way to stay up to date with the latest updates of this great project?

### DISCUSSION THREAD

**Discussion by virtualistic** [vmware/packer-examples-for-vsphere#642] [Help]
First off: Thank you @tenthirtyam Ryan for creating this great project! I had a simple packer setup in the past for my homelab to quickly get VMs up and running but time took its toll and updates in various components broke that setup some how. Your project is just what the doctor ordered!!!

What I was wondering: What would be the best way to stay up to date in the future with any changes that are made to the project?
I git cloned the current version (23.01) but when you do updates to the scripts or menu, or I change things (for example, change the time zone or swap out CentOS with Oracle linux for example)
What would be the easiest way to "do my own thing" while finding a way to get the latest additions made to the project?
Thanks for reading!

**Comment by tenthirtyam** [comment]
- https://github.com/vmware-samples/packer-examples-for-vsphere/issues/695
